website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md

```diff
@@ -232,7 +232,7 @@ To set up Airflow and dbt Cloud, you can:
 
 ![airflow dbt run select](/img/blog/2021-11-29-dbt-airflow-spiritual-alignment/airflow-connection-ID.png)
 
-3. Set up your Airflow DAG similar to [this example](https://github.com/apache/airflow/blob/main/airflow/providers/dbt/cloud/example_dags/example_dbt_cloud.py).
+3. ~Set up your Airflow DAG similar to this example.~
```

Comment:
```suggestion
3. Set up your Airflow DAG similar to this example.
```

Reply:
question: which example is step 3 referring to? is it the PNG graph referenced on line 239 below?

Reply:
it doesn't render the strikethrough, which is odd: 

https://deploy-preview-2315--docs-getdbt-com.netlify.app/blog/dbt-airflow-spiritual-alignment

<img width="932" alt="Screen Shot 2022-10-26 at 1 38 55 PM" src="https://user-images.githubusercontent.com/107218380/198133300-85cbd2bc-0059-42ec-b89f-23d06b6d38ff.png">